packages/react-components/src/organisms/SpeakersList.tsx

```diff
@@ -0,0 +1,149 @@
+    gap: 0,
+  },
+});
+
+const userStyles = css({
+  overflow: 'hidden',
+  display: 'grid',
+  gridTemplateColumns: 'min-content 1fr',
+  gridColumnGap: `${6 / perRem}em`,
+  alignItems: 'center',
+});
+
+const gridMixin = {
+  display: 'grid',
+  [`@media (min-width: ${tabletScreen.width}px)`]: {
+    gridTemplateColumns: '1fr 1fr 1fr',
+    gridAutoFlow: 'column',
+    alignItems: 'start',
+  },
+};
+
+const speakerListStyles = css({
+  ...gridMixin,
+  [`@media (max-width: ${mobileScreen.width}px)`]: {
+    gridAutoFlow: 'row',
+    alignItems: 'start',
+    borderBottom: '1px solid #DFE5EA',
+  },
+});
+
+const headerStyle = css({
+  ...gridMixin,
+  [`@media (max-width: ${mobileScreen.width}px)`]: {
+    display: 'none',
+  },
+});
+
+const labelStyle = css({
+  [`@media (max-width: ${mobileScreen.width}px)`]: {
+    gridTemplateColumns: '1fr 1fr 1fr',
+    gridAutoFlow: 'column',
+    alignItems: 'start',
+    display: 'inline-block',
+    fontWeight: 'bold',
+  },
+  display: 'none',
+});
+
+const groupStyle = css({
+  display: 'flex',
+  flexFlow: 'column',
+  [`@media (max-width: ${mobileScreen.width}px)`]: {
+    paddingBottom: `${20 / perRem}em`,
+    paddingTop: `${20 / perRem}em`,
+  },
+});
+
+const toBeAnnouncedStyle = css({
+  fontStyle: 'italic',
+});
+
+interface SpeakerListProps {
+  speakers: EventSpeaker[];
+  readonly endDate: string;
+}
+
+const SpeakerList: React.FC<SpeakerListProps> = ({ speakers, endDate }) => {
+  const hasEnded = useDateHasPassed(considerEndedAfter(endDate));
+  const userToBeAnnounced = hasEnded
+    ? 'User was never announced'
+    : 'To be announced';
+
+  return (
+    <div>
+      <Headline3 styleAsHeading={3}>Speakers</Headline3>
+      <div css={headerStyle}>
+        <Headline4 styleAsHeading={4}>Team</Headline4>
+        <Headline4 styleAsHeading={4}>Speaker</Headline4>
+        <Headline4 styleAsHeading={4}>Role</Headline4>
+      </div>
+      <div css={gridStyles}>
+        {speakers.map(({ user, team, role }, index) => (
+          <div key={`speaker-id-${index}`} css={speakerListStyles}>
+            <div css={groupStyle}>
+              <div css={labelStyle}>
+                <span>Team</span>
+              </div>
+              <Link href={network({}).teams({}).team({ teamId: team.id }).$}>
+                {team.displayName}
+              </Link>
+            </div>
+            <div css={groupStyle}>
+              <div css={labelStyle}>
+                <span>Speaker</span>
+              </div>
+              <div css={userStyles}>
+                <Avatar
+                  {...user}
+                  imageUrl={
+                    user?.displayName
+                      ? user.avatarUrl || ''
+                      : getPlaceholderAvatarUrl()
+                  }
+                />
+                {(user?.displayName && (
+                  <Link
+                    href={
+                      user && network({}).users({}).user({ userId: user?.id }).$
```

Comment:
Not sure why we have a ? in user.id and what happens for external users who don't have a profile